vpr/src/device/rr_graph/rr_graph_fwd.h

```diff
@@ -0,0 +1,74 @@
+/**********************************************************
+ * MIT License
+ *
+ * Copyright (c) 2018 LNIS - The University of Utah
+ *
+ * Permission is hereby granted, free of charge, to any person obtaining a copy
+ * of this software and associated documentation files (the "Software"), to deal
+ * in the Software without restriction, including without limitation the rights
+ * to use, copy, modify, merge, publish, distribute, sublicense, and/or sell
+ * copies of the Software, and to permit persons to whom the Software is
+ * furnished to do so, subject to the following conditions:
+ *
+ * The above copyright notice and this permission notice shall be included in all
+ * copies or substantial portions of the Software.
+ *
+ * THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+ * IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+ * FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
+ * AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+ * LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+ * OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE
+ * SOFTWARE.
+ ***********************************************************************/
+
+/************************************************************************
+ * Filename:    rr_graph_fwd.h
+ * Created by:   Xifan Tang
+ * Change history:
+ * +---------------------------------------------------------+
+ * |  Date       |    Author     | Notes
+ * +---------------------------------------------------------+
+ * | 2017/11/21  |  Kevin Murray | Created as a prototype 
+ * +---------------------------------------------------------+
+ * | 2019/07/31  |  Xifan Tang   | Complete an alpha version
+ * +---------------------------------------------------------+
+ ***********************************************************************/
```

Comment:
Similarly we don't keep revision history in the file comments. This information should be fully available in revision control which will never become out of date.

Reply:
The comments have been removed. See latest commit.

Reply:
Removed